gatsby-node.js

```diff
@@ -43,36 +43,54 @@ exports.createPages = ({ graphql, actions }) => {
 
       const slugArray = slug.split('/');
 
+      // extract the version from the slug
+      let version = Array.from(slug.matchAll(/^\/docs\/([a-zA-Z0-9\.]*)\//g)).map(val => val[1])[0];
```

Comment:
Why not using match?

Reply:
made me reconsider.
exec on a regex is alright for this :) shorter to express.